src/components/CreateWallet.jsx

```diff
@@ -60,53 +60,439 @@ const WalletCreationForm = ({ createWallet, isCreating }) => {
+    }
+  }, [
+    wordOne,
+    wordTwo,
+    wordThree,
+    wordFour,
+    wordFive,
+    wordSix,
+    wordSeven,
+    wordEight,
+    wordNine,
+    wordTen,
+    wordEleven,
+    wordTwelve,
+    seedphrase,
+  ])
 
   return (
     <div>
-      <div className="mb-4">
-        <div>Wallet Name</div>
-        <div className="fs-4">{walletDisplayName(createdWallet.name)}</div>
-      </div>
-      <div className="mb-4">
-        <Seedphrase seedphrase={createdWallet.seedphrase} isBlurred={!revealSensitiveInfo} />
-      </div>
-      <div className="mb-4">
-        <div>Password</div>
-        <div className={`fs-4${revealSensitiveInfo ? '' : ' blurred-text'}`}>
-          {!revealSensitiveInfo ? 'randomrandom' : createdWallet.password}
+      <div className="fs-4">Confirm seed phrase backup</div>
+      <p className="text-secondary">Enter each word in sequential order.</p>
+
+      <rb.Form noValidate>
+        <div className="container">
+          <div className="row mb-4">
+            <div className="col">
+              <rb.InputGroup>
+                <rb.InputGroup.Text>1.</rb.InputGroup.Text>
+                <rb.FormControl
+                  type="text"
+                  placeholder="Word 1"
+                  value={wordOne}
+                  onChange={(e) => {
+                    setWordOne(e.target.value)
+                  }}
+                  disabled={wordOne === seedphrase[0]}
+                  isInvalid={wordOne !== seedphrase[0] && wordOne.length > 0}
+                  isValid={wordOne === seedphrase[0]}
+                  required
+                />
+                {wordOne === seedphrase[0] && !seedBackup && (
+                  <rb.Form.Control.Feedback>Word matches!</rb.Form.Control.Feedback>
+                )}
+                {wordOne !== seedphrase[0] && wordOne.length > 0 && (
+                  <rb.Form.Control.Feedback type="invalid">Word does not match.</rb.Form.Control.Feedback>
+                )}
+              </rb.InputGroup>
+            </div>
+            <div className="col">
+              <rb.InputGroup>
+                <rb.InputGroup.Text>2.</rb.InputGroup.Text>
+                <rb.FormControl
+                  type="text"
+                  placeholder="Word 2"
+                  value={wordTwo}
+                  onChange={(e) => {
+                    setWordTwo(e.target.value)
+                  }}
+                  disabled={wordTwo === seedphrase[1]}
+                  isInvalid={wordTwo !== seedphrase[1] && wordTwo.length > 0}
+                  isValid={wordTwo === seedphrase[1]}
+                  required
+                />
+                {wordTwo === seedphrase[1] && !seedBackup && (
+                  <rb.Form.Control.Feedback>Word matches!</rb.Form.Control.Feedback>
+                )}
+                {wordTwo !== seedphrase[1] && wordTwo.length > 0 && (
+                  <rb.Form.Control.Feedback type="invalid">Word does not match.</rb.Form.Control.Feedback>
```

Comment:
Thinking about it again, we could drop the text feedback. That would prevent the inputs jumping around when feedback appears / disappears. The red / green outline is probably enough of a feedback anyway. Thoughts?

Reply:
That would make it even cleaner, will implement.